tests/getRaw.py

```diff
@@ -25,8 +25,9 @@
 import os
 
 import unittest
-import lsst.utils.tests as utilsTests
+import lsst.utils
```

Comment:
in this case it might be clearer to just have both of those `lsst.utils` imports since they are not equivalent and it would clarify what was a real substantive part of the patch from what is collateral damage.